Host/Mdis_Test/Mdis_Test.sh

```diff
@@ -345,6 +345,8 @@ function cleanMdisTestBackgroundJob {
                 echo "${LogPrefix} Could not kill cat backgroung process ${MdisTestBackgroundPID}"
             else
                 echo "${LogPrefix} process ${MdisTestBackgroundPID} killed"
+                wait ${MdisTestBackgroundPID}
+                MdisTestBackgroundPID=0
             fi
             sleep 1
```

Comment:
Probably we wont need this sleep 1 as the wait call will do that sleep properly, right? However, I don't think it hurts leaving the sleep 1. Just saying.